flask_api.py

```diff
@@ -108,8 +109,14 @@ def handle_question():
     if "question" not in request_body:
         return "request body should include the question", BAD_REQUEST
 
+    # Store
+    initializeDB()
```

Comment:
call to `initializeDB()` is unnecessary here because of the `initializeNimbus()` call above.